typed-racket-test/succeed/with-integer-linear-arith2.rkt

```diff
@@ -16,7 +16,7 @@
 (ann (vector 0 1 2 3 4 5 6 7) (V8 Any))
 (ann (vector 0 1 2 3 4 5 6 7) (V8 Byte))
 (ann '#(0 1 2 3 4 5 6 7) (V8 Any))
-(ann '#(0 1 2 3 4 5 6 7) (V8 Byte))
+(ann '#(0 1 2 3 4 5 6 7) (Refine [v : (Immutable-Vector Byte Byte Byte Byte Byte Byte Byte Byte)] (= 8 (vector-length v))))
```

Comment:
Maybe I'm reading this wrong... but for the "given" to be a subtype of the "expected", wouldn't `Integer` have to be a subtype of `Byte`?